package/android/src/main/java/com/mrousavy/camera/core/CameraSession.kt

```diff
@@ -575,8 +575,9 @@ class CameraSession(
       val videoStabilizationMode = videoStabilizationMode
       val lowLightBoost = lowLightBoost
       val hdr = hdr
+      val enableTorch = enableTorch
```

Comment:
Btw, is there a reason to redeclare the variables here?
Just being curious since I never coded in kotlin!

Reply:
Yea it had something to do with threading previously, but I think now it could be removed. I'll do that in a later PR :)